### PR TITLE
Fix for issue #76, nested relative paths for unit test location

### PR DESCRIPTION
**Pull request by johnpaliotta**
There are two cases handled in the fix
- An illegal value for the path, like: "./unitTests/****"
- A valid value with nested syntax like: "./unittests/a/b/c

@Zbigor both should be tested

### CODE REVIEW

**Comment by johnpaliotta** [comment]
@aytey Igor created the test and is passes, any comments on the changes here, or can we merge?

File: src/vcastTestInterface.ts [hunk 1]
```diff
@@ -653,8 +653,16 @@ function configureWorkspaceAndBuildEnviro(
...
       .then((answer) => {
         if (answer === "Yes") {
-          fs.mkdirSync(unitTestLocation);
-          commonNewEnvironmentStuff(fileList, unitTestLocation);
+          try {
+            fs.mkdirSync(unitTestLocation, { recursive: true });
+            commonNewEnvironmentStuff(fileList, unitTestLocation);
+          } catch (error) {
+            vscode.window.showErrorMessage(
+              `Error creating directory: ${unitTestLocation}, update the 'Unit Test Location' option to a valid value`
```
Comment (aytey) [inline src/vcastTestInterface.ts:661]: Is there a chance there's anything in `error` that we should be showing to the user? For example, if it is a permissions problem?

Comment (johnpaliotta) [reply src/vcastTestInterface.ts:661]: Good point, I had not tested a permissions error, dialog looks like this now

image

Comment (johnpaliotta) [reply src/vcastTestInterface.ts]: Nope

**Pull request merged**